packages/react-ui/src/theme/sections/typography.js

```diff
@@ -9,117 +11,219 @@ export const themeTypography = {
+    fontSize: getPixelToRem(11),
+    lineHeight: 1.454,
+    letterSpacing: '0.2px'
   },
   overline: {
-    fontFamily: '"Open Sans", sans-serif',
+    fontFamily: '"Overpass Mono", monospace',
+    fontWeight: 600,
+    fontSize: getPixelToRem(10),
+    lineHeight: 1.2,
+    letterSpacing: '2px',
+    textTransform: 'uppercase'
+  }
+};
+
+const customTypography = {
+  body1Italic: {
+    ...baseTypography.body1,
+    fontStyle: 'italic'
+  },
+  body1Strong: {
+    ...baseTypography.body1,
+    fontWeight: 600
+  },
+  body1StrongItalic: {
+    ...baseTypography.body1,
+    fontWeight: 600,
+    fontStyle: 'italic'
+  },
+  body2Italic: {
+    ...baseTypography.body2,
+    fontStyle: 'italic'
+  },
+  body2Strong: {
+    ...baseTypography.body2,
+    fontWeight: 600
+  },
+  body2StrongItalic: {
+    ...baseTypography.body2,
+    fontWeight: 600,
+    fontStyle: 'italic'
+  },
+  buttonLarge: {
+    fontFamily: 'Inter, sans-serif',
+    fontWeight: 500,
+    fontSize: getPixelToRem(16),
+    lineHeight: 1.5,
+    letterSpacing: '0.25px'
+  },
+  buttonSmall: {
+    fontFamily: 'Inter, sans-serif',
+    fontWeight: 500,
+    fontSize: getPixelToRem(11),
+    lineHeight: 1.454,
+    letterSpacing: '0.4px'
+  },
+  captionItalic: {
+    ...baseTypography.caption,
+    fontStyle: 'italic'
+  },
+  captionStrong: {
+    ...baseTypography.caption,
+    fontWeight: 500
+  },
+  captionStrongItalic: {
+    ...baseTypography.caption,
+    fontWeight: 500,
+    fontStyle: 'italic'
+  },
+  overlineDelicate: {
+    ...baseTypography.overline,
+    fontWeight: 400,
+    letterSpacing: '1.2px'
+  },
+  codeLarge: {
+    fontFamily: '"Overpass Mono", monospace',
+    fontWeight: 400,
+    fontSize: getPixelToRem(16),
+    lineHeight: 1.5,
+    letterSpacing: 0
+  },
+  codeLargeStrong: {
+    fontFamily: '"Overpass Mono", monospace',
+    fontWeight: 600,
+    fontSize: getPixelToRem(16),
+    lineHeight: 1.5,
+    letterSpacing: 0
+  },
+  codeMedium: {
+    fontFamily: '"Overpass Mono", monospace',
     fontWeight: 400,
-    fontSize: '0.625rem',
-    lineHeight: 1.6,
-    letterSpacing: '0.150em',
-    textTransform: 'uppercase',
-    fontSmoothing: 'antialiased'
+    fontSize: getPixelToRem(14),
+    lineHeight: 1.428,
+    letterSpacing: 0
+  },
+  codeMediumStrong: {
+    fontFamily: '"Overpass Mono", monospace',
+    fontWeight: 600,
+    fontSize: getPixelToRem(14),
+    lineHeight: 1.428,
+    letterSpacing: 0
+  },
+  codeSmall: {
+    fontFamily: '"Overpass Mono", monospace',
+    fontWeight: 400,
+    fontSize: getPixelToRem(12),
+    lineHeight: 1.333,
+    letterSpacing: 0
+  },
+  codeSmallStrong: {
+    fontFamily: '"Overpass Mono", monospace',
+    fontWeight: 600,
+    fontSize: getPixelToRem(12),
+    lineHeight: 1.333,
+    letterSpacing: 0
+  },
+  codeSmallChip: {
+    fontFamily: '"Overpass Mono", monospace',
+    fontWeight: 600,
+    fontSize: getPixelToRem(12),
+    lineHeight: 1.25,
+    letterSpacing: 0
   },
   charts: {
```

Comment:
Probably it's ok as a first step, but I would probably suggest treating this for charts in a separated object. Ideally this would land finally in something like a charts.typography (kind-of-internal) section in the future, related to charts and probably not exposed externally (thus no need to include in theme.typography.charts...).

I know this is a challenge now, but I'm sharing thoughts for future direction, we can advance step by step to avoid breaking current charts

. At least let's add a note

Reply:
I've talked to them yesterday about this chart variant, because is not in the new palette (this is correct) but we don't have a replacement for it (we need one because is too specific to a particular component).

They told me that they are going to create a new one (reusable) that can replace that, so I left a TODO in the code with this situation. I didn't want no move it until we have the replacement, to see where it belongs.

Is this enough or do you think I should do something different?